src/Database/Drivers/SqlsrvDriver.php

```diff
@@ -91,15 +91,28 @@ public function formatLike($value, $pos)
 	 */
 	public function applyLimit(& $sql, $limit, $offset)
 	{
-		if ($limit >= 0) {
-			$sql = preg_replace('#^\s*(SELECT|UPDATE|DELETE)#i', '$0 TOP ' . (int) $limit, $sql, 1, $count);
-			if (!$count) {
-				throw new Nette\InvalidArgumentException('SQL query must begin with SELECT, UPDATE or DELETE command.');
+		$info = $this->connection->query("SELECT CAST(SERVERPROPERTY('ProductVersion') AS varchar) AS [ver]")->fetch();
```

Comment:
Is possible to use `$connection->getPdo()->getAttribute(\PDO::ATTR_SERVER_VERSION)` like in MySqlDriver?